libsolidity/analysis/ControlFlowBuilder.cpp

```diff
@@ -608,6 +663,52 @@ bool ControlFlowBuilder::visitNode(ASTNode const& _node)
 	return true;
 }
 
+void ControlFlowBuilder::analyzeFunctionCall(FunctionCall const& _functionCall)
```

Comment:
Can you rename this function ta make it clear that it checks that the function being called always reverts?

Reply:
I'm still of the opinion, that this should be done as an additional analysis step after we already have the graph...

Reply:
If we want to do this in an extra step, we need to save in each CFGNode the function calls it did that require further analysis.. so I would need to extend the struct with e.g. `vector<FunctionCall const*> possiblyRevertingFunctionCalls` or something like that